app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/exemptionandlosses/annual_exempt_amount.scala.html

```diff
@@ -25,7 +25,9 @@
 @import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.finance.MoneyUtils
 @import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.returns.DisposalDate
 @import uk.gov.hmrc.cgtpropertydisposalsfrontend.views.components.AutoCompleteType
-
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.UserType
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.UserType
```

Comment:
looks like you've got a double import here. Did the compiler not give a warning?